doc/tutorials/python/01-lennard_jones/01-lennard_jones.tex

```diff
@@ -129,7 +129,7 @@ \section{Introduction}
 
 \section{Background}
 Today's research on Soft Condensed Matter has brought the needs for having a 
-flexible, extensible, reliable and efficient (parallel) molecular simulation 
+flexible, extensible, reliable, and efficient (parallel) molecular simulation 
```

Comment:
It's ambigious on whether or not to use the oxford comma for adjectives. It's not necessary in these cases, but one should advocate consistency.